runtime/compilers/rillv1/parse_rillyaml.go

```diff
@@ -64,7 +64,7 @@ type rillYAML struct {
 	// Default YAML values for migrations
 	Migrations yaml.Node `yaml:"migrations"`
 	// Feature flags
-	Features []string `yaml:"features"`
+	Features map[string]bool `yaml:"features"`
```

Comment:
Do we have users using a feature flag today? Because this is a breaking change, and a parse error in `rill.yaml` can break the entire project.

For backwards compatibility, you could use `yaml.Node` and try parsing it as both a map and an array :)

Reply:
Agreed with Brian that we'll make this change to maintain backwards compatibility